src/main/java/com/sapient/model/beans/Expense.java

```diff
@@ -48,6 +48,10 @@ public Integer getId() {
 //		this.title = title;
 //	}
 
+	public Boolean inMonth(MonthType month, Integer year){
+		return date.getMonth() == month.ordinal() && (date.getYear()+1900) == year;
```

Comment:
I just looked into this, perhaps the weirdest java thing I have seen ever. It is actually a deprecated function but I couldn't quickly find a replacement so yolo